src/writeData/components/ClientLibrarySection.tsx

```diff
@@ -44,15 +53,24 @@ const ClientLibrarySection = () => {
         Back-end, front-end, and mobile applications
       </Heading>
       <SquareGrid cardSize="170px" gutter={ComponentSize.Small}>
-        {items.map(item => (
-          <WriteDataItem
-            key={item.id}
-            id={item.id}
-            name={item.name}
-            image={item.logo}
-            url={`${CLIENT_LIBS}/${item.id}`}
-          />
-        ))}
+        {items.map(item => {
```

Comment:
Should we pull this out into a function or component since it's been written 3 times in 3 components?

Reply:
there's some weirdness where its sometimes items.logo and sometimes items.image, or how the urls are all different. that strange distinction coupled with how simple it is led me to believe that these snippets dont necessarily do the same thing, but very similar things. if it was more complicated, enough to validate normalizing the input and abstracting the link info out of it, i think they would totally warrant some reusability patterns, but for this case it seems like the line count to support the reusability aspect is pretty close to just copying